templates/flavors/base/cluster-template.yaml

```diff
@@ -126,6 +126,8 @@ spec:
           diskSizeGB: 256
           lun: 0
       sshPublicKey: ${AZURE_SSH_PUBLIC_KEY}
+      kubeadmConfigSpec:
```

Comment:
Here's the code change, the remaining diff is generated code

Reply:
I pointed you to the wrong file earlier, this is the one you want https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/templates/flavors/default/machine-deployment.yaml#L47

sorry for the confusion

Reply:
np go generate is doing all the work here :)